### PR TITLE
[KYUUBI  #6283] Improve the message of distribution built without enabling web ui

### LINKED ISSUE #6283: [TASK][TRIVIAL] Improve message of distribution built without enabling web ui
### What's the level of this task?

TRIVIAL

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before creating

- [X] I have searched in the [task list](https://github.com/orgs/apache/projects/343) and found no similar tasks.

### Mentor

- [ ] I have sufficient expertise on this task, and I volunteer to be a mentor of this task to guide contributors through the task.

### Skill requirements

- How to build and run Kyuubi
- Basic HTML and English writing skills

### Background and Goals

The building of Web UI is optional, user must enable `--web-ui` on creating distributions when using `build/dist` tool to create binary distributions, e.g. 
```
./build/dist --tgz --web-ui --spark-provided --flink-provided --hive-provided
``` 

Otherwise, if you access the web ui, only the following dummy page shows.

image

### Implementation steps

Enhance `kyuubi-server/src/main/resources/dist/index.html` to improve the message.

### Additional context

Introduction of [2024H1 Kyuubi Code Contribution Program](https://github.com/apache/kyuubi/issues/6232)

### PR DESCRIPTION
**Pull request by dupen01**
# :mag: Description
Improve message of distribution built without enabling web ui.
## Issue References 🔗

This pull request fixes #6283 

## Describe Your Solution 🔧

Here is my implement:

image

## Types of changes :bookmark:

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

#### Related Unit Tests

---

# Checklist 📝

- [ ] This patch was not authored or co-authored using [Generative Tooling](https://www.apache.org/legal/generative-tooling.html)

**Be nice. Be informative.**

### CODE REVIEW

**Review by cxzl25 — APPROVED** [review]

File: kyuubi-server/src/main/resources/dist/index.html [hunk 1]
```diff
@@ -23,6 +23,11 @@
     <title>Apache Kyuubi Dashboard</title>
   </head>
   <body>
-    <div>This is a dummy page for development.</div>
+    <h2 style="text-align: center;">🙁 The Web UI is currently unavailable.</h2>
+    <h4 style="text-align: center;">👉 If you wish to access the Web UI,
+    you must enable `--web-ui` when using `build/dist` tool to create binary distributions, e.g.</h4>
```
[the remark this answers is not in the document]
Comment (dupen01) [reply kyuubi-server/src/main/resources/dist/index.html:28]: ok. I will revise it and prepare a new one.

**Review by pan3793 — APPROVED** [review]

**Comment by dupen01** [comment]
> LGTM, please update the screenshot

Sure. The screenshot has been updated.

**Comment by pan3793** [comment]
Thanks, merged to master/1.9/1.8

**Comment by pan3793** [comment]
@dupen01 you may want to link your email dunett@163.com to your GitHub account, otherwise, your contribution will not be recognized by GitHub

**Pull request closed without merging**